src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!--  <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="frame">
+        <legend>
+          Personal information
+        </legend>
+        <label class="field">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+            name="lastName"
+          >
+        </label>
+        <label class="field">
+          Name:
+          <input
+            type="text"
+            name="firstName"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="field">
+          Full date of birth:
+          <input
+            type="date"
+            name="years"
+            value="12"
+          >
+        </label>
+        <label class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+          >
+        </label>
+        </value>
+      </fieldset>
+      <fieldset class="frame">
+        <legend>
+          Registration
+        </legend>
+        <label class="field">
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="frame">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <label class="field">
+          Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="answer"
+            value="Yes"
+          >
+          <label for="yes">
```

Comment:
Fix the labels here. It doesn't work correctly for now 

https://github.com/mate-academy/layout_html-form/assets/99403198/2e7e02d8-009d-41df-ba91-6f82f9002c52

Reply:
Hi @Anastasiia-Svintsova about this part I do not understand what I have to do? It's giving only one choice yes or no, do I have to change both to be selected? Please explain to me and thanks for your suggestions :)